test/unified_memory_allocation/uma_pools/disjoint_pool_config.cpp

```diff
@@ -0,0 +1,325 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: MIT
+
+#include "helpers.hpp"
+#include "usm_pool_config.hpp"
+
+using PoolDescriptor = usm_pool::PoolDescriptor;
+
+const PoolDescriptor DESCRIPTORS[4] = {PoolDescriptor(UR_USM_TYPE_HOST),
```

Comment:
Make it a map instead (or unordered_map), with keys like "HOST_POOL_DESCRIPTOR" etc., and values being actual PoolDescriptors. Key name from a map explains better what its value is comparing to DESCRIPTORS[i] call.

Reply:
Done.